src/config.ts

```diff
@@ -29,13 +23,11 @@ export class Config {
   network: { name: string; chainId: number };
   registrar: { address: string; domain: string };
   radToken: { address: string; faucet: string };
-  orgFactory: { address: string };
   reverseRegistrar: { address: string };
-  orgs: { subgraph: string; contractHash: string; pinned: string[] };
+  orgs: { contractHash: string };
```

Comment:
contactHash isn't needed either afaict

Reply:
It's still used by `utils.identifyAddress` which in turn is used by `Address.svelte`.